### PR TITLE
add hugo default TOC

### LINKED ISSUE #2846: ToC documentation links not working
**What steps did you take and what happened:**

The Table of Contents links at the top of documentation pages no longer work, eg:

- https://velero.io/docs/v1.4/basic-install/
- https://velero.io/docs/v1.4/customize-installation/

Looking at the source you can see the `href` is blank, eg: `<a href="">Basic Install</a>`

**What did you expect to happen:**

The links used to work and navigate to the specified location on page.

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

### PR DESCRIPTION
**Pull request by a-mccarthy**
Signed-off-by: Abigail McCarthy <mabigail@vmware.com>

fixes #2846

### CODE REVIEW

**Comment by a-mccarthy** [comment]
This updates the TOC for main and 1.4 docs. How far back should we support for docs?

**Comment by ashish-amarnath** [comment]
I think we can make these changes in main and the 1.5-beta and keep it going forward.
changes to 1.4 docs may not be necessary.

**Comment by nrb** [comment]
@a-mccarthy I'd say `main` and `v1.4` for now, since technically `v1.4` is the latest version.

**Comment by nrb** [comment]
Looks like this has a conflict that needs to be resolved in the `start-contributing.md` file.

**Comment by a-mccarthy** [comment]
@nrb i think i fixed the merge conflict?

@ashish-amarnath I think we are ok to just have main and 1.4 updated, since pre-1.5 is more temporary and the 1.5 docs will get the changes from main.

File: site/content/docs/v1.4/debugging-restores.md [hunk 1]
```diff
@@ -3,9 +3,6 @@ title: "Debugging Restores"
 layout: docs
 ---
 
-* [Example][0]
```
Comment (ashish-amarnath) [inline site/content/docs/v1.4/debugging-restores.md:6]: same comment on the file from `main` version

File: site/content/docs/main/debugging-restores.md [hunk 2]
```diff
@@ -3,9 +3,6 @@ title: "Debugging Restores"
 layout: docs
 ---
 
-* [Example][0]
```
[the remark this answers is not in the document]
Comment (a-mccarthy) [reply site/content/docs/main/debugging-restores.md:6]: removed :)

**Pull request merged**